src/contract/userop/contract.ts

```diff
@@ -0,0 +1,141 @@
+
+import { ethers, utils, ContractInterface, Signer } from "ethers";
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
+import { VerifiedWallet } from "../../wallet";
+import { Client, Presets, UserOperationMiddlewareFn } from "userop";
+
+export enum USEROP_DATATYPES {
+    VALUE = 'number',
+    STRING = 'string',
+    ADDRESS = 'address',
+}
+
+const response: { transactionHash: string | null, result: UserOperationEventEvent | null } = { transactionHash: null, result: null }
+export type UseropResponse = typeof response;
+
+
+export class UseropVerifiedContract {
```

Comment:
@herberthk 

Do not change VerifiedContract because the userops related config parameters can all be exported to be used by callContract. Changing VerifiedContract will break all the sdk contracts.